python_modules/dagster/dagster/_core/execution/context/compute.py

```diff
@@ -1,1907 +1,16 @@
-from abc import ABC, ABCMeta, abstractmethod
 from contextlib import contextmanager
 from contextvars import ContextVar
 from inspect import _empty as EmptyAnnotation
-from typing import AbstractSet, Any, Dict, Iterator, List, Mapping, Optional, Sequence, Union, cast
+from typing import Iterator, Optional, Union
```

Comment:
i always found the name of this file confusing - maybe now that its just the context manager maybe it could be like `enter_context_helper` or something else. not beholden to this though

Reply:
The way I had interpreted it was that "compute context"  is the more general term that spans OpExecutionContext, AssetExecutionContext, and AssetCheckExecutionContext.  I don't feel particularly invested in preserving this, but I do think `enter_context_helper` would be a little narrow, given that this module also holds `ExecutionContextTypes` and `current_execution_context`.

In light of that, going to punt figuring out the right name to a future PR.